src/test/groovy/BaseSpec.groovy

```diff
@@ -1,17 +1,43 @@
+import DemoQA.AlertsFramesAndWindowsPage
+import DemoQA.BrowserWindowsPage
+import DemoQA.PracticeFormPage
+import DemoQA.SelectMenuPage
+import IFrame.IFramePage
+import TablePress.TablePress
 import geb.spock.GebReportingSpec
 
 class BaseSpec extends GebReportingSpec {
 
-    void setup(){
+    void setup() {
         browser.driver = browser.config.driver
         browser.go()
     }
 
-    void cleanup(){
+    void cleanup() {
         driver.quit()
     }
 
-    void navigateToPracticeFormPage(){
+    void navigateToPracticeFormPage() {
```

Comment:
You can create a method that accepts a page URL and navigates to it so that we don't have a separate method for each page.